src/components/ui/selection-not-found.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Image
+ * A wrapper around image elements to handle loading states and
+ * transitions
+ */
```

Comment:
Is this a particular format that produces docs/comments/etc in this library, or is this just a preferred style?

Reply:
Just preferred style at the time of writing. Actually this needs to be removed. I copy/pasted the crud out of this.

Reply:
Eliminated